packages/rocketchat-ui-account/client/accountPreferences.html

```diff
@@ -101,76 +101,76 @@ <h1>{{_ "Messages"}}</h1>
 								</div>
 							</div>
 							<div class="input-line double-col" id="hideUsernames">
 								<label>{{_ "Hide_usernames"}}</label>
 								<div>
-									<label><input type="radio" name="hideUsernames" value="1" checked="{{checked 'hideUsernames' true}}" /> {{_ "True"}}</label>
-									<label><input type="radio" name="hideUsernames" value="0" checked="{{checked 'hideUsernames' false}}" /> {{_ "False"}}</label>
+									<label><input type="radio" name="hideUsernames" value="true" checked="{{checked 'hideUsernames' true}}" /> {{_ "True"}}</label>
+									<label><input type="radio" name="hideUsernames" value="false" checked="{{checked 'hideUsernames' false}}" /> {{_ "False"}}</label>
 								</div>
 							</div>
 							{{#if showRoles}}
 								<div class="input-line double-col" id="hideRoles">
 									<label>{{_ "Hide_roles"}}</label>
 									<div>
-										<label><input type="radio" name="hideRoles" value="1" checked="{{checked 'hideRoles' true}}" /> {{_ "True"}}</label>
-										<label><input type="radio" name="hideRoles" value="0" checked="{{checked 'hideRoles' false}}" /> {{_ "False"}}</label>
+										<label><input type="radio" name="hideRoles" value="true" checked="{{checked 'hideRoles' true}}" /> {{_ "True"}}</label>
+										<label><input type="radio" name="hideRoles" value="false" checked="{{checked 'hideRoles' false}}" /> {{_ "False"}}</label>
 									</div>
 								</div>
 							{{/if}}
 							<div class="input-line double-col" id="hideFlexTab">
 								<label>{{_ "Hide_flextab"}}</label>
 								<div>
-									<label><input type="radio" name="hideFlexTab" value="1" checked="{{checked 'hideFlexTab' true}}" /> {{_ "True"}}</label>
-									<label><input type="radio" name="hideFlexTab" value="0" checked="{{checked 'hideFlexTab' false}}" /> {{_ "False"}}</label>
+									<label><input type="radio" name="hideFlexTab" value="true" checked="{{checked 'hideFlexTab' true}}" /> {{_ "True"}}</label>
+									<label><input type="radio" name="hideFlexTab" value="false" checked="{{checked 'hideFlexTab' false}}" /> {{_ "False"}}</label>
 								</div>
 							</div>
 							<div class="input-line double-col" id="hideAvatars">
 								<label>{{_ "Hide_Avatars"}}</label>
 								<div>
-									<label><input type="radio" name="hideAvatars" value="1" checked="{{checked 'hideAvatars' true}}"/> {{_ "True"}}</label>
-									<label><input type="radio" name="hideAvatars" value="0" checked="{{checked 'hideAvatars' false}}"/> {{_ "False"}}</label>
+									<label><input type="radio" name="hideAvatars" value="true" checked="{{checked 'hideAvatars' true}}"/> {{_ "True"}}</label>
+									<label><input type="radio" name="hideAvatars" value="false" checked="{{checked 'hideAvatars' false}}"/> {{_ "False"}}</label>
 								</div>
 							</div>
-							<div class="input-line double-col">
+							<!-- <div class="input-line double-col">
```

Comment:
Remove html comments

Reply:
We will use this html when we implement that feature